src/main/java/se/iths/crimedatabase/security/SecurityConfig.java

```diff
@@ -11,22 +11,26 @@
 @Configuration
 @EnableWebSecurity
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
-
-    private final String[] urls = {"/addresses", "/categories", "/crimes", "/criminals", "/users", "/victims", "/publish"};
-
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http
-                .csrf()
-                .ignoringAntMatchers(urls)
+                .csrf().disable()
+                .authorizeRequests()
+                .antMatchers("/images/**").permitAll()
```

Comment:
Why do we want to perimit everyone to acces everything in images? Seems unsafe if we want to add something later on